trace/kafka/component_test.go

```diff
@@ -0,0 +1,201 @@
+package kafka
+
```

Comment:
tests look good ! Just a suggestion ... 
Would it make sense also to asses the value we provide on the cfg property (instead of just the builder property) ?
In the end, thats where we want to put the corresponding value ...

Reply:
I've added assertions that the `cfg` property is set for the three fields we're setting values; the Version, the Timeout and the RequiredAcks policy, both in the separate and grouped tests. Thanks!